src/main/java/LittleMissHelpful.java

```diff
@@ -0,0 +1,241 @@
+                        System.out.println(lineBreak + "\nHai-yo, task number must be an integer lah.\n" + lineBreak);
+                    } catch (TaskNotFoundException e) {
+                        System.out.println(e.getMessage());
+                    }
+                    continue;
+
+                } else if (command.equalsIgnoreCase("unmark")) {
+                    try {
+                        int listIndex = Integer.parseInt(item) - 1;
+                        if (listIndex < 0 || listIndex >= counter) {
+                            throw new TaskNotFoundException(lineBreak + "Hai-ya, task number out of range lah.\n" + lineBreak);
+                        }
+                        Task curTask = list.get(listIndex);
+                        list.set(listIndex, curTask.unmarkTask());
+                        System.out.println(lineBreak);
+                        System.out.println("Ok, I see you laze. You have marked this task as not done yet: " + curTask.toString());
+                        System.out.println(lineBreak);
+                    } catch (NumberFormatException e) {
+                        System.out.println(lineBreak + "\nHai-yo, task number must be an integer lah.\n" + lineBreak);
+                    } catch (TaskNotFoundException e) {
+                        System.out.println(e.getMessage());
+                    }
+                    continue;
+                }
+
+                //deletes task from list
+                if (command.equalsIgnoreCase("delete")) {
+                    try {
+                        int listIndex = Integer.parseInt(item) - 1;
+                        if (listIndex < 0 || listIndex >= counter) {
+                            throw new TaskNotFoundException(lineBreak + "\nHai-ya, task number out of range lah.\n" + lineBreak);
+                        }
+                        Task curTask = list.get(listIndex);
+                        list.remove(listIndex);
+                        counter--;
+
+                        System.out.println(lineBreak);
+                        System.out.println("Wah shiok! This task no more liao: " + curTask.toString());
+                        System.out.println("Now got only " + counter + " tasks left.");
+                        System.out.println(lineBreak);
+                    } catch (NumberFormatException e) {
+                        System.out.println(lineBreak + "\nHai-yo, task number must be an integer lah.\n" + lineBreak);
+                    } catch (TaskNotFoundException e) {
+                        System.out.println(e.getMessage());
+                    }
+                    continue;
+                }
+
+                // Adds tasks to list
+                if (command.equals("todo") || command.equals("deadline") || command.equals("event")) {
+                    Task newTask = null;
+
+                    try {
+                        if (command.equalsIgnoreCase("todo")) {
```

Comment:
The switch and case statement may come in handy here. (frankly I also never use)